### PR TITLE
New package: sublime-music-0.12.0

### PR DESCRIPTION
**Pull request by elbachir-one**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl

Addresses: #53103

### CODE REVIEW

File: srcpkgs/sublime-music/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+changelog="https://github.com/sublime-music/sublime-music/blob/master/CHANGELOG.rst"
+distfiles="https://github.com/sublime-music/sublime-music/archive/refs/tags/v${version}.tar.gz"
+checksum=faaf102fd790ecea3d966c9d6e305567c734a1df30eaa6bbdedf93d863bc7435
+
+do_check() {
+	pytest
+}
+
+post_install() {
+	pip install --break-system-packages deepdiff python-mpv dataclasses-json
```
Comment (classabbyamp) [inline srcpkgs/sublime-music/template:23]: if

Comment (classabbyamp) [inline srcpkgs/sublime-music/template:23]: this should never be done. if you're missing dependencies, make templates for them too

File: srcpkgs/sublime-music/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+short_desc="Native Subsonic/Airsonic/sonic client for Linux built with Python/GTK+"
+maintainer="El Bachir <bachiralfa@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://sublimemusic.app"
+changelog="https://github.com/sublime-music/sublime-music/blob/master/CHANGELOG.rst"
+distfiles="https://github.com/sublime-music/sublime-music/archive/refs/tags/v${version}.tar.gz"
+checksum=faaf102fd790ecea3d966c9d6e305567c734a1df30eaa6bbdedf93d863bc7435
+
+do_check() {
+	pytest
```
Comment (classabbyamp) [inline srcpkgs/sublime-music/template:19]: this is what the default does, no need to override it

File: srcpkgs/sublime-music/template (hunk 1 above)
Comment (elbachir-one) [reply srcpkgs/sublime-music/template:23]: If the dependencies are not in the Void Linux repositories and can only be installed using pip or manually, do I need to create templates for them as well?

File: srcpkgs/sublime-music/template (hunk 2 above)
Comment (elbachir-one) [reply srcpkgs/sublime-music/template:19]: Okay.

**Comment by elbachir-one** [comment]
**Note:** Some dependencies need to be installed manually using pip (templates have not been created for them yet). Run the following command: `pip3 install deepdiff dataclasses-json`

**Pull request closed without merging**